### DISCUSSION TITLE
Which version of copier to use?

### DISCUSSION THREAD

**Discussion by ghost** [pawamoy/copier-pdm#29] [Q&A]
I followed the document and installed latest copier (8.1.0) by pipx.  
After I run `copier copy .........`, copier says `Template uses unsafe feature: jinja_extensions`. (I'm sure that I executed (`pipx inject copier copier-templates-extensions`)  
I tried rollback to copier 8.0.0, the same.  
Then I rollbacked to copier 7.2.0, then copier say the templates needs copier>=8.  
So...... which version should I use?

**Comment by pawamoy** [comment]
Ah yes, sorry, I forgot to update the docs. Copier 8 introduced a `--trust` or `--UNSAFE` flag to allow using Jinja extensions. You just need to pass it to the copy command:

```
copier copy --trust gh:pawamoy/copier-pdm
```

Upgrade Copier again to >=8, pass this flag, and you should be good to go :slightly_smiling_face:
